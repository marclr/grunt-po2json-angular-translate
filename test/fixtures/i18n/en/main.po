msgid ""
msgstr ""

msgid "HELLO_WORLD"
msgstr "Hello world"

msgid "INTRO"
msgstr "Introduction"

msgid "DESCRIPTION"
msgstr "Description"
