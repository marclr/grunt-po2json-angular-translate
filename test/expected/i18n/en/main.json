{
   "HELLO_WORLD": "Hello world",
   "INTRO": "Introduction",
   "DESCRIPTION": "Description"
}